applications/MeshMovingApplication/custom_elements/structural_meshmoving_element.cpp

```diff
@@ -117,7 +118,7 @@ StructuralMeshMovingElement::SetAndModifyConstitutiveLaw(
                          // elements; 0 = no stiffening
   const double quotient = factor / detJ0[PointNumber];
   const double weighting_factor = detJ0[PointNumber] * std::pow(quotient, xi);
-  const double poisson_coefficient = 0.3;
+  const double poisson_coefficient = this->pGetProperties()->GetValue(POISSON_RATIO);
```

Comment:
```suggestion
  const double poisson_coefficient = this->pGetProperties()->Has(POISSON_RATIO) ? this->pGetProperties()->GetValue(POISSON_RATIO) : 0.3;
```

Reply:
I don't like this because it is a second place where the default value is given.